cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -82,3 +86,49 @@ def is_ansible_pod_stopped(enabled_embedded_appliance):
         assert wait_for(is_nginx_stopped, func_args=[enabled_embedded_appliance], num_sec=30)
     else:
         assert wait_for(is_ansible_pod_stopped, func_args=[enabled_embedded_appliance], num_sec=180)
+
+
+@pytest.mark.tier(1)
+def test_embed_tower_event_catcher_process(appliance, enabled_embedded_appliance):
+    """
+    EventCatcher process is started after Ansible role is enabled (rails
+    evm:status)
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/4h
+        tags: ansible_embed
+    """
+    result = appliance.ssh_client.run_command(
+        "cd /var/www/miq/vmdb; rake evm:status | grep 'EmbeddedAnsible'"
+    ).output
+
+    checks = [
+        "EmbeddedAnsible",
+        "EmbeddedAnsible::Automation::EventCatcher",
+        "EmbeddedAnsible::Automation::Refresh",
+    ]
+
+    for data in result.splitlines():
+        for i in checks:
+            if i in data:
+                assert "started" in data
+
+
+@pytest.mark.tier(1)
+def test_embed_tower_logs(appliance, enabled_embedded_appliance):
+    """
+    Separate log files should be generated for Ansible to aid debugging.
+    p1 (/var/log/tower)
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/4h
+        tags: ansible_embed
+    """
+    log_status = appliance.ssh_client.run_command("ls /var/log/tower/")
+    assert log_status.success
```

Comment:
Optional:
This check is ok... but for `logs` check... I think we can also check that after deleting log file like `var/log/tower/tower.log` it should recreate after activity.

This is my personal observation for `evm.log`. let me know if you like to add some other checks.

Reply:
I will add soft_asserts for `logs` under `tower` directory. When you enable Ansible server role only once it will have log file created there is no such activity present.

Reply:
@digitronik Instead of `soft_assert` I have added `assert` because every log is necessary.